doc/hardware/at86rf212.rst

```diff
@@ -0,0 +1,105 @@
+.. at86rf212:
+
+AT86RF212 setup
+##################
+
+Category: radio
+Standard: IEEE 802.15.4
+Boards: MinnowBoard MAX, Galileo Gen 2
+
+Connection with MinnowBoard MAX board
+
+
+.. image:: _static/at86rf212-minnow.png
+
+Connection with Galileo Gen 2 board
+
+
+.. image:: _static/at86rf212-galileo.png
+
+
+.. table:: Pin table of at86rf212 radio board:
+
+    =====  ==========================
+    PIN    Name
+    =====  ==========================
+    1    	SLP_TR
+    2    	CLKM
+    3    	DIG2
+    4    	VCC
+    5    	DIG1
+    6    	VCC
+    7    	SCLK
+    8    	GND
+    9    	nRST
+    10    	GND
+    11    	IRQ
+    12    	MISO
+    13    	DIG4
+    14    	MOSI
+    15		DIG3
+    16		NSEL
+    =====  ==========================
+
+
+.. table:: Connect at86rf212 board to MinnowBoard MAX low speed expansion connector (LSE):
```

Comment:
While a picture might be nice showing this wiring layout, these tables clearly show what connects to what and work just fine for me.